tests/integration/errors/test_error_handler_implicit.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should NOT capture and report error types  
+when a custom error handler exists but it returns true for that error type.
+*/
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "7.0", "<")) {
+  die("skip: PHP < 7.0.0 not supported\n");
+}
+*/
+
+/*INI
+display_errors=1
+log_errors=0
+*/
+
+/*EXPECT_REGEX
+Let this serve as a warning
+*/
+
+/*EXPECT_TRACED_ERRORS null */
+
+/*EXPECT_ERROR_EVENTS null */
+
+function errorHandlerOne($errno, $errstr, $errfile, $errline)
+{
+    switch ($errno) {
+        case E_USER_DEPRECATED:
+            return;
+        }
+    return false;
+}
+
+// set to the user defined error handler
+$old_error_handler = set_error_handler("errorHandlerOne");
+
+trigger_error("Let this serve as a deprecation", E_USER_DEPRECATED);  
+
+echo("Let this serve as a warning");         
```

Comment:
I think this message has wrong connotations. This message is from a section of a program that didn't generate any errors, warnings or exception. This is from a normal execution of a program. Just seeing 'warning' in expected output, when actually nothing alarming happened raises an eyebrow.
```suggestion
echo("Hello from happy path!");
```

Reply:
Please see: [8d432b4](https://github.com/newrelic/newrelic-php-agent/pull/869/commits/8d432b4c8ef3e5066bb3badfb2d3a456b6cf8efc)

Reply:
Thank you! :)